sams/sampler/NvidiaSMI.py

```diff
@@ -166,12 +166,15 @@ def do_sample(self):
 
     def sample(self):
         logger.debug("sample()")
+        self._most_recent_sample = []
```

Comment:
```suggestion
        most_recent_sample = []
```
I just remembered that there will be a race condition here, so we need one edit here...